Code/autopkglib/__init__.py

```diff
@@ -78,8 +78,10 @@ def is_linux():
         kCFPreferencesCurrentHost,
     )
 except:
-    log("WARNING: Failed 'from Foundation import NSArray, NSDictionary' in " + __name__)
-    log(
+    print(
```

Comment:
These were converted back to print() because log() isn't defined yet when this import runs.